sql-plugin/src/main/scala/com/nvidia/spark/rapids/ColumnarPartitionReaderWithPartitionValues.scala

```diff
@@ -114,4 +178,70 @@ object ColumnarPartitionReaderWithPartitionValues {
       }
     }
   }
+
+  // scalastyle:off line.size.limit
```

Comment:
Not sure why this is here, as I don't see anything below that couldn't be wrapped if necessary.